zallet/src/components/json_rpc/methods/list_unified_receivers.rs

```diff
@@ -33,8 +37,29 @@ pub(crate) struct ListUnifiedReceivers {
 
 pub(super) const PARAM_UNIFIED_ADDRESS_DESC: &str = "The unified address to inspect.";
 
-pub(crate) fn call(unified_address: &str) -> Response {
-    warn!("TODO: Implement z_listunifiedreceivers({unified_address})");
+pub(crate) fn call(wallet: &DbConnection, unified_address: &str) -> Response {
+    let address = match UnifiedAddress::decode(wallet.params(), unified_address) {
+        Ok(addr) => addr,
+        Err(e) => return Err(LegacyCode::InvalidParameter.with_message(e.to_string())),
```

Comment:
The `zcashd` RPC returns `LegacyCode::InvalidAddressOrKey` if it is not any kind of Zcash address, and `LegacyCode::InvalidParameter` if it is a known Zcash address but not a Unified Address. The way we'd emulate this is by parsing with `ZcashAddress` (from the `zcash_address` crate) to get the first error, and then trying to convert into a UA to get the second error.

Either make that change, or add a TODO so that we don't forget to fix this later.

(The error message string should also match `zcashd` if we want to further reduce the semantic difference, but getting the error codes to match where we can is more important.)